src/github/pullRequestOverview.ts

```diff
@@ -391,7 +391,8 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
 	}
 	private getAssigneesQuickPickItems(
 		assignableUsers: IAccount[] | undefined,
-		suggestedReviewers: ISuggestedReviewer[] | undefined,
+		suggestedReviewers: IAccount[] | undefined,
```

Comment:
I think this is an old copy-paste error, but are there actually any callers of `getAssigneesQuickPickItems` that will pass undefined for these parameters now?

Reply:
You are right, I removed the undefined types